### PR TITLE
Recent sessions component UI fixes

### PR DESCRIPTION
**Pull request by bmuniswamy**
# Critical Changes

# Changes
    - Moved the individual card to sessionCard component. 
    - Added status condition to the ServiceSessionSelector class so we only show Pending and Complete status. 
    - Added NavigationMixin so when the sessionName is clicked it takes the user to the 
      serviceSession record View page where the attendance component will reside.
# Issues Closed

# New Metadata

# Deleted Metadata

# Definition of Done
  Refer to [Asteroids DoD document](https://salesforce.quip.com/iq2mAy4i62oM) to see any additional details for the items below
~~- [ ] Place holder data is incorporated (Refer to [Case Management Sample Data document](https://quip.com/cbFcAPJF8t0z))~~
~~- [ ] Base axe-core JEST test included for any new LWC (No critical violations)~~
~~- [ ] CRUD/FLS is enforced in Apex~~
- [x] Permission sets & field sets are updated to account for CRUD/FLS/TABS for new object metadata
- [x] All modified classes are unit tested (Apex) at 100% coverage
- [x] UX approval or UX not necessary
- [x] PR contains draft release notes
- [x] Attach work item to the pull request with [Lurch](https://salesforce.quip.com/50ZRA5LEWVzH) `**Lurch:attach W-000000` or `**Lurch:add`
- [ ] All **acceptance criteria** have been met
    - [x] Developer
    - [ ] Code Reviewer
    - [ ] QA
- [ ] QE story level testing completed

### CODE REVIEW

**Comment by bmuniswamy** [comment]
[W-8091293](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008IsNZIA0/view)

**Review by bethbrains — CHANGES_REQUESTED** [review]
A few small changes requested. Great work!

File: force-app/main/default/lwc/recentSessions/recentSessions.html [hunk 1]
```diff
@@ -4,9 +4,29 @@
         <div
             class="slds-border_bottom slds-var-p-left_small slds-var-p-bottom_small slds-var-p-top_xxx-small"
         >
-            <h1 slot="title" class="header-title-container slds-card__header-title">
-                {labels.recentSessions}
+            <h1
+                slot="title"
+                class="header-title-container slds-card__header-title slds-text-title_bold"
+            >
+                <span class="slds-card__header-title">{labels.recentSessions}</span>
```
[the remark this answers is not in the document]
Comment (bmuniswamy) [reply force-app/main/default/lwc/recentSessions/recentSessions.html:11]: Had to remove the css class from the h1 tag. We will need the span here

File: force-app/main/default/lwc/recentSessions/recentSessions.js [hunk 2]
```diff
@@ -71,32 +78,28 @@ export default class RecentSessions extends LightningElement {
...
-                              " " +
-                              this.objectLabelPlural,
-                });
+            for (const [key, value] of Object.entries(result.data)) {
+                let menuItem = {};
+                menuItem.label = value;
+                menuItem.value = key;
+
+                //Set the default value
+                if (key === THIS_WEEK) {
```
[the remark this answers is not in the document]
Comment (bmuniswamy) [reply force-app/main/default/lwc/recentSessions/recentSessions.js:97]: I have declared this as a constant on line 20. I think that is why the label audit did not fail here

File: force-app/main/default/classes/ServiceSessionSelector.cls [hunk 3]
```diff
@@ -9,6 +9,7 @@
 public with sharing class ServiceSessionSelector {
     public List<ServiceSession__c> getServiceSessionsByStartDate(String dateLiteral) {
         List<ServiceSession__c> allSessions = new List<ServiceSession__c>();
+        Set<String> sessionStatus = new Set<String>{ 'Pending', 'Complete' };
```
[the remark this answers is not in the document]
Comment (bmuniswamy) [reply force-app/main/default/classes/ServiceSessionSelector.cls:12]: Renamed it to `allowedSessionStatues`

**Review by bethbrains — APPROVED** [review]
Looks good!

**Review by lbeller — APPROVED** [review]

**Comment by bmuniswamy** [comment]
@lbeller, I have gone ahead and made the changes for the accordion toggle labels that were in a PR that was closed. Can you please look at it and approve the PR.

**Review by lbeller — APPROVED** [review]

**Pull request merged**